spec/cucumber/formatter/legacy_api/adapter_spec.rb

```diff
@@ -1562,7 +1562,7 @@ def find_after_step_hooks(test_case)
               end
             end
             expect( formatter.legacy_messages ).to eq([
-                :before_features,
+                                                        :before_features,
                   :before_feature,
```

Comment:
This occurs multiple times in the same file; I think [this comment](https://github.com/cucumber/cucumber-ruby/pull/1045#discussion_r85471144) refers to the issue. 
Do you wish me to fix them all manually?